misc/ls-extensions/modules/project-design-service/src/main/java/io/ballerina/projectdesign/generators/entity/EntityModelGenerator.java

```diff
@@ -56,13 +57,17 @@ public class EntityModelGenerator {
 
     private final SemanticModel semanticModel;
     private final ComponentModel.PackageId packageId;
+    private final String moduleRootPath;
 
-    public EntityModelGenerator(SemanticModel semanticModel, ComponentModel.PackageId packageId) {
+    public EntityModelGenerator(SemanticModel semanticModel, ComponentModel.PackageId packageId,
+                                String moduleRootPath) {
```

Comment:
Any specific reason to use `String` instead of `Path` as path variable type?

Reply:
yes, `Path` is the most suitable type here.